beginner_source/quickstart/build_model_tutorial.py

```diff
@@ -0,0 +1,153 @@
+# ----------------------------
+#
+
+import os
+import torch
+import torch.nn as nn
+import torch.onnx as onnx
+from torch.utils.data import DataLoader
+from torchvision import datasets, transforms
+
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+print('Using {} device'.format(device))
+
+# model
+model = nn.Sequential(
+        nn.Flatten(),
+        nn.Linear(28*28, 512),
+        nn.ReLU(),
+        nn.Linear(512, 512),
+        nn.ReLU(),
+        nn.Linear(512, len(classes)),
+        nn.Softmax(dim=1)
+    ).to(device)
+    
+print(model)
+
+#############################################
+# Class nn.Module Example:
+# --------------------------
+#
+
+class NeuralNework(nn.Module):
+    def __init__(self, x):
+        super(NeuralNework, self).__init__()
+        self.layer1 = nn.Linear(28*28, 512)
+        self.layer2 = nn.Linear(512, 512)
+        self.output = nn.Linear(512, 10)
+
+    def forward(self, x):
+        x = F.relu(self.layer1(x))
+        x = F.relu(self.layer2(x))
+        x = self.output(x)
+        return F.softmax(x, dim=1)
+
+#############################################
+# Get Device for Training
+# -----------------------
+# Here we check to see if `torch.cuda <https://pytorch.org/docs/stable/notes/cuda.html>`_ is available to use the GPU, else we will use the CPU. 
+#
+# Example:
+#
+
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+print('Using {} device'.format(device))
+
+##############################################
+# The Model Module Layers
+# -------------------------
+#
+# Lets break down each model layer in the FashionMNIST model.
+#
+
+##################################################
+# `nn.Flatten <https://pytorch.org/docs/stable/generated/torch.nn.Flatten.html>`_ to reduce tensor dimensions to one.
+# -----------------------------------------------
+#
+# From the docs:
+# ``torch.nn.Flatten(start_dim: int = 1, end_dim: int = -1)``
+#
+
+# Here is an example using one of the training_data set items:
+tensor = training_data[0][0]
+print(tensor.size())
+
+# Output: torch.Size([1, 28, 28])
+
+model = nn.Sequential(
+    nn.Flatten()
+)
+flattened_tensor = model(tensor)
+flattened_tensor.size()
+
+# Output: torch.Size([1, 784])
+
+##############################################
+# `nn.Linear <https://pytorch.org/docs/stable/generated/torch.nn.Linear.html>`_ to add a linear layer to the model.
+# -------------------------------
+#
+# Now that we have flattened our tensor dimension we will apply a linear layer transform that will calculate/learn the weights and the bias.
+#
+# From the docs:
+# 
+# ``torch.nn.Linear(in_features: int, out_features: int, bias: bool = True)``
+#
+
+input = training_data[0][0]
+print(input.size())
+model = nn.Sequential(
+    nn.Flatten(),    
+    nn.Linear(28*28, 512),
+)
+output = model(input)
+output.size()
+
+
+# Output: 
+# torch.Size([1, 28, 28])
+# torch.Size([1, 512])
+
+#################################################
+# Activation Functions
```

Comment:
This section looks disparate from the above. 

I feel like we may not need a whole section with examples for the layers (the API docs has examples). Instead we could add a comment in-line describing what the layer is, and briefly hint at why it is used here along with a hyperlink to the API doc